packages-backend/loggers/src/formatters/rewrite-fields.ts

```diff
@@ -9,16 +9,31 @@ type TRewriteField = {
   from: string;
   to: string;
   replaceValue?: (value: unknown) => unknown;
+  unsetFromField?: boolean;
 };
 type TRewriteFieldsFormatterOption = {
   fields: TRewriteField[];
 };
 
+const defaultRewriteField: TRewriteField = {
+  unsetFromField: true,
+};
 function rewriteField(info: TransformableInfo, field: TRewriteField) {
-  const val = getIn(info, field.from);
-  if (val) {
-    unsetIn(info, field.from);
-    setIn(info, field.to, field.replaceValue ? field.replaceValue(val) : val);
+  const mergedRewriteField = { ...defaultRewriteField, ...field };
```

Comment:
This is needed as as `field: TRewriteField = defaultRewriteField` would not work as defaulted parameters are not merged when they are objects.

Reply:
Do we need to merge here? Since it's only one option maybe we can simply inline the condition `field.keepOriginalField ?? false`?

Reply:
Would work too. Usually I find merging more scalable. Will adjust here.